Slider/Slider.js

```diff
@@ -267,60 +87,263 @@ const SliderBase = kind({
 				handleDecrement
 			])
 		),
+
 		onKeyUp: handle(
 			forProp('disabled', false),
 			forwardWithPrevent('onKeyUp'),
 			forProp('activateOnSelect', true),
 			forKey('enter'),
 			forward('onActivate')
 		)
-	},
-
-	computed: {
-		className: ({activateOnSelect, active, showAnchor, styler}) => styler.append({
-			activateOnSelect,
-			active,
-			showAnchor
-		}),
-		knobStep: validateSteppedOnce(props => props.knobStep, {
-			component: 'Slider',
-			stepName: 'knobStep',
-			valueName: 'max'
-		}),
-		step: validateSteppedOnce(props => props.step, {
-			component: 'Slider',
-			valueName: 'max'
-		}),
-		tooltip: ({tooltip}) => tooltip === true ? ProgressBarTooltip : tooltip
-	},
-
-	render: ({css, disabled, focused, tooltip, ...rest}) => {
-		delete rest.activateOnSelect;
-		delete rest.active;
-		delete rest.onActivate;
-		delete rest.knobStep;
-		delete rest.showAnchor;
-
-		return (
-			<UiSlider
-				{...rest}
-				aria-disabled={disabled}
-				css={css}
-				disabled={disabled}
-				progressBarComponent={
-					<ProgressBar css={css} />
-				}
-				tooltipComponent={
-					<ComponentOverride
-						component={tooltip}
-						css={css}
-						visible={focused}
-					/>
-				}
-			/>
-		);
-	}
-});
+	}, props, spotlightAccelerator);
+
+	const componentClassName = classnames(
+		componentCss.slider,
+		className,
+		{
+			[componentCss.activateOnSelect]: activateOnSelect,
```

Comment:
Maybe this is a missing footprint to be removed. The original code has this class due to a renaming from `activateOnFocus` in #143, but the PR also removed the class from LESS file.